app/src/main/java/com/example/campusguide/Bootstrapper.kt

```diff
@@ -115,5 +89,9 @@ class Bootstrapper constructor(activity: MapsActivity) {
         // Drawer
         val drawer = Drawer(activity, login)
         drawer.setupDrawer()
+
+        // Bottom Navigation
+        val bottomNavigation = BottomNavigation(activity, map, permissions, locationProvider)
+        bottomNavigation.setupBottomNavigation()
```

Comment:
Why not put the logic for setupBottomNavigation in the constructor of BottomNavigation?